src/components/EmbeddedPDF.tsx

```diff
@@ -13,25 +17,32 @@ type TProps = {
 };
 
 const EmbeddedPDF = ({ document, documentPassageMatches = [], passageIndex = null, startingPassageIndex = 0 }: TProps) => {
+  const isLoading = useState(true);
   const containerRef = useRef(null);
   const adobeKey = useContext(AdobeContext);
 
-  const pdfPreview = usePDFPreview(document, documentPassageMatches, adobeKey);
+  const pdfPreview = usePDFPreview(document, adobeKey);
 
-  // Ensure the instance of the PDF client is not reset on render
+  // Ensure the instance of the PDF client is not reset on re-render
   // otherwise we lose the ability to interact with the pdf
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  const { createPDFClient, passageIndexChangeHandler } = useMemo(() => pdfPreview, [document, documentPassageMatches, adobeKey]);
+  const { createPDFClient, passageIndexChangeHandler, documentMatchesChangeHandler } = useMemo(() => pdfPreview, [document, adobeKey]);
 
   useEffect(() => {
     if (containerRef?.current !== null) {
-      createPDFClient(startingPassageIndex);
+      createPDFClient();
     }
-  }, [containerRef, document, createPDFClient, startingPassageIndex]);
+  }, [containerRef, document, createPDFClient]);
 
   useEffect(() => {
-    passageIndexChangeHandler(passageIndex);
-  }, [passageIndexChangeHandler, passageIndex]);
+    passageIndexChangeHandler(passageIndex, documentPassageMatches);
+    /* trunk-ignore(eslint/react-hooks/exhaustive-deps) */
```

Comment:
suggestion: Could we leave a description as to why we do this? I assume it's because we pass a string of the object, rather than the object? 

Why is that?